modules/phase_field/examples/anisotropic_interfaces/snow_adapt.i

```diff
@@ -0,0 +1,165 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 16
+  ny = 16
+  xmax = 9
+  ymax = 9
+  elem_type = QUAD4
+[]
+
+[Adaptivity]
+  initial_marker = marker
+  initial_steps = 3
+  max_h_level = 3
+  marker = marker
+  [./Markers]
+    [./marker]
+      type = ErrorFractionMarker
+      refine = 0.85
+      coarsen = 0.1
+      indicator = indicator
+    [../]
+  [../]
+  [./Indicators]
+    [./indicator]
+      type = GradientJumpIndicator
+      variable = w
+    [../]
+  [../]
+[]
+
+[Variables]
+  [./w]
+  [../]
+  [./T]
+    initial_condition = 0.0
+  [../]
+[]
+
+[ICs]
+  [./wIC]
+    type = SmoothCircleIC
+    variable = w
+    block = 0
+    int_width = 0.01
+    x1 = 4.5
+    y1 = 4.5
+    radius = 0.5
+    outvalue = 0
+    invalue = 1
+    3D_spheres = false
+  [../]
+[]
+
+[AuxVariables]
+  [./local_energy]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+  [./m]
+    order = CONSTANT
+    family = MONOMIAL
+  [../]
+[]
+
+[Kernels]
+  [./w_dot]
+    type = TimeDerivative
+    variable = w
+  [../]
+  [./anisoACinterface1]
+    type = ACInterfaceKobayashi1
+    variable = w
+    mob_name = M
+  [../]
+  [./anisoACinterface2]
+    type = ACInterfaceKobayashi2
+    variable = w
+    mob_name = M
+  [../]
+  [./AllenCahn]
+    type = AllenCahn
+    variable = w
+    mob_name = M
+    f_name = fbulk
+    args = m
+  [../]
+  [./T_dot]
+    type = TimeDerivative
+    variable = T
+  [../]
+  [./CoefDiffusion]
+    type = Diffusion
+    variable = T
+  [../]
+  [./w_dot_T]
+    type = CoefCoupledTimeDerivative
+    variable = T
+    v = w
+    coef = -1.8
+  [../]
+[]
+
+[AuxKernels]
+  [./m]
+    type = ParsedAux
+    variable = m
+    args = T
+    constant_names = pi
+    constant_expressions = 3.14159265359
```

Comment:
Cool - then I'd suggest that. It help train people moving forward to define expressions instead of some random number of digits or precision for things like this.